flow/benchmarks/baselines/bottleneck0.py

```diff
@@ -39,90 +28,64 @@ def bottleneck0_baseline(num_runs, render=True):
         SumoExperiment
             class needed to run simulations
     """
+    exp_tag = flow_params["exp_tag"]
+    sumo_params = flow_params["sumo"]
+    env_params = flow_params["env"]
+    net_params = flow_params["net"]
```

Comment:
I'm not sure I agree with these changes; this significantly reduces readability. Additionally, you may want to use a longer horizon on the baselines to ensure that the results are accurate